modules/deploying-models-using-multiple-gpu-nodes.adoc

```diff
@@ -23,22 +23,15 @@ endif::[]
 * You have downloaded and installed the {openshift-platform} command-line interface (CLI). See link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/cli_tools/openshift-cli-oc#installing-openshift-cli[Installing the OpenShift CLI^].
 
 ifndef::upstream[]
-* You have enabled the operators for your GPU type, such as Node Feature Discovery Operator, NVIDIA GPU Operator, and so on. For more information about enabling accelerators, see link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/working_with_accelerators[Working with accelerators^].
+* You have enabled the operators for your GPU type, such as Node Feature Discovery Operator, NVIDIA GPU Operator. For more information about enabling accelerators, see link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/working_with_accelerators[Working with accelerators^].
 endif::[]
 ifdef::upstream[]
-* You have enabled the operators for your GPU type, such as Node Feature Discovery Operator, NVIDIA GPU Operator, and so on. For more information about enabling accelerators, see link:{odhdocshome}/working-with-accelerators[Working with accelerators^].
+* You have enabled the operators for your GPU type, such as Node Feature Discovery Operator, NVIDIA GPU Operator. For more information about enabling accelerators, see link:{odhdocshome}/working-with-accelerators[Working with accelerators^].
 endif::[]
 
-** You are using one of the following GPU types: `nvidia.com/gpu` (default), `intel.com/gpu`, `amd.com/gpu`, `habana.ai/gaudi`, or you have specified another GPU type in the annotations of the `InferenceService`. 
+** You are using an NVIDIA GPU (`nvidia.com/gpu`).
 ** Specify the GPU type through either the `ServingRuntime` or `InferenceService`. If the GPU type specified in the `ServingRuntime` differs from what is set in the `InferenceService`, both GPU types are assigned to the resource and can cause errors. 
-
-ifndef::upstream[]
-* You have set up model-serving in raw deployment mode. For more information about raw deployment, see link:{rhoaidocshome}{default-format-url}serving_models/serving-large-models_serving-large-models#deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode_serving-large-models[Deploying models on single node {openshift-platform} using KServe raw deployment mode]
-endif::[]
-ifdef::upstream[]
-* You have set up model-serving in raw deployment mode.
-endif::[]
-
+* You have specified `serving.kserve.io/deploymentMode: RawDeployment` in the `InferenceService`. Your cluster does not have to be fully configured for KServe raw deployment.
```

Comment:
@rnetser @Jooho 
Does this update work here?

Reply:
maybe rephrase to `make sure your cluster is configured with kserve enabled`?